src/Microsoft.VisualStudio.Threading/JoinableTask.cs

```diff
@@ -53,13 +53,11 @@ public partial class JoinableTask : IJoinableTaskDependent
     /// <summary>
     /// The <see cref="JoinableTaskDependencyGraph.JoinableTaskDependentData"/> to track dependencies between tasks.
     /// </summary>
-    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
```

Comment:
I didn't mind your removing the attribute from the internal `JoinableTaskDependencyGraph` struct, but removing them from common, public types like this will expose complex internal details to the average user, which ain't great. Can we get a before and after screenshot of the impact this has on the debugging experience for JoinableTask and JoinableTaskCollection?

Reply:
The new fields are highlighted:

JoinableTaskCollection:
![image](https://user-images.githubusercontent.com/1103906/203685919-5f683ba1-b504-448c-bc95-67c428ce0659.png)


JoinableTask:
![image](https://user-images.githubusercontent.com/1103906/203685620-7dfd83d0-4970-4df6-94ae-54e1294cc89c.png)

Both are needed to diagnose common hangs with JTF.